server/cmd/code-annotation/handler/assignments.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+
+	"github.com/src-d/code-annotation/server/cmd/code-annotation/serializer"
+	"github.com/src-d/code-annotation/server/model"
+	"github.com/src-d/code-annotation/server/repository"
+)
+
+func HandleGetAssignmentsForUserExperiment(w http.ResponseWriter, r *http.Request) {
```

Comment:
do you think we could follow https://github.com/golang/go/wiki/CodeReviewComments#package-names here?
The package has name `handler`. So I would suggest `handler.GetAssignmentsByUser`. I think we don't really need `experiment` in the name because looks like assignments always belong to experiment. Feel free to suggest better name!

Reply:
I'd sugest `handler.HandleInHandlerPackageThatGetAssignmentsByUserExperiment`
:trollface: 

You're totally right.

I only disagree in the `Experiment` suffix, because there will be many experiments; that's because handled URL is `/experiments/{experimentId}/assignments`.